### DISCUSSION TITLE
How can I swap ALT and WINDOWS key for a specific keyboard?

### DISCUSSION THREAD

**Discussion by wthe2004** [hyprwm/Hyprland#5128] [Q&A] [answered]
How can I swap ALT and WINDOWS key for a specific keyboard? I only find the way to switch Caps and ESC in the official documentation. Thanks in advance for any help!

**Comment by CalfMoon** [comment]
`kb_options = altwin:swap_alt_win` swaps alt and windows.

**Reply by kicre** [reply]
How to make the two sets coexist?
I tried both of the following and neither of them worked.
```
kb_options = altwin:swap_alt_win
kb_options = caps:swapescape
```
```
kb_options = altwin:swap_alt_win & caps:swapescape
```

**Reply by fufexan** [reply]
Have you tried with `,` ?

**Reply by kicre** [reply]
> Have you tried with `,` ?您尝试过 `,` 吗？

"," seems to have other effects in Hyprlang. I just tried it and didn't work.

**Comment by avisek** [comment, marked as answer]
Complete answer for those who haven't yet figured out:

```diff
 input {
   # ...
   
+  kb_options = altwin:swap_alt_win
 }
```

**Comment by vincentaxhe** [comment]
kmonad can do anything about remap keyboard
